pallets/parachain-system/src/migration.rs

```diff
@@ -44,6 +45,13 @@ pub fn on_runtime_upgrade<T: Config>() -> Weight {
 	weight
 }
 
+pub struct Migration<T: Config>(PhantomData<T>);
```

Comment:
Moving them to individual structs is a good change, but then you also need to remove `on_runtime_upgrade` in the pallets.

Reply:
ok I was not sure about that, what happens if you keep both?
I will also need to make a PR to xcm then

Reply:
Also please add docs to all these migration structs.

Reply:
✅